babble/build.gradle

```diff
@@ -34,7 +34,7 @@ apply plugin: io.mosaicnetworks.getbabblecore.GetBabbleCorePlugin
 // def repository = "https://github.com/mosaicnetworks/babble/releases/download"
 // def method = "https"
```

Comment:
This needs to be moved to a released version. Does not need to be part of this Pull Request, but does need to be done today.

Reply:
Yes, it will be done as part of the release process.